webapp/sources/rudder/rudder-web/src/main/webapp/secure/administration/policyServerManagement.html

```diff
@@ -506,6 +506,127 @@ <h3 class="page-title">Logging</h3>
       </div>
     </div>
 
+    <div class="inner-portlet">
+      <h3 class="page-title">Policy Generation System Hooks</h3>
+      <div class="portlet-content">
+        <div class="lift:administration.PropertiesManagement.generationHookCfpromise" id="generationHookCfpromise">
+          <div class="col-xs-12 callout-fade callout-warning">
+            <div class="marker">
+              <span class="glyphicon glyphicon-info-sign"></span>
+            </div>
+            <h4>Development Policy Check</h4>
+            <p>Rudder can check generated policies syntax after generation to ensure that non syntactly valide policies are deployed on node.</p>
```

Comment:
typo: non syntactically valid policies

Reply:
and the sentence by itself means the opposite

Rudder can check generated policies syntax after generation to ensure that only syntactically valid policies are deployed on node.